server.js

```diff
@@ -1311,29 +1311,36 @@ cache(function(data, match, sendBadge, request) {
 // Bountysource integration.
 camp.route(/^\/bountysource\/team\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var team = match[1];  // eg, `mozilla-core`.
-  var type = match[2];  // eg, `activity`.
-  var format = match[3];
-  var url = 'https://api.bountysource.com/teams/' + team;
-  var options = {
+  const team = match[1];  // eg, `mozilla-core`.
+  const type = match[2];  // eg, `activity`.
```

Comment:
Why is `type` actually needed here? By looking at the code below, it seems as if `activity` is the only value we would be expecting, so maybe we should hardcode it into the path above and simplify/optimise the code a tiny little bit. 😃

Reply:
Yeah - I wasn't sure about this. Doing a bit more code archaeology, I think the idea was that additional types could be added like this: https://github.com/badges/shields/pull/307 but the only PR to do so never went anywhere so lets just hard-code it and simplify.